src/groundlight/client.py

```diff
@@ -500,26 +530,35 @@ def ask_async(
             assert image_query.id is not None
 
             # Do not attempt to access the result of this query as the result for all async queries
-            # will be None. Your result is being computed asynchronously and will be available
-            # later
+            # will be None. Your result is being computed asynchronously and will be available later
             assert image_query.result is None
 
-            # retrieve the result later or on another machine by calling gl.get_image_query()
-            # with the id of the image_query above
-            image_query = gl.get_image_query(image_query.id)
+            # retrieve the result later or on another machine by calling gl.wait_for_confident_result()
+            # with the id of the image_query above. This will block until the result is available.
+            image_query = gl.wait_for_confident_result(image_query.id)
 
             # now the result will be available for your use
             assert image_query.result is not None
 
+            # alternatively, you can check if the result is available (without blocking) by calling
+            # gl.get_image_query() with the id of the image_query above. This will return the image_query,
+            # but the result may still be None.
```

Comment:
```suggestion
            # but the result may still be UNCLEAR.
```
possible fix...but not the only option